test/run-test.ps1

```diff
@@ -20,9 +20,9 @@ else {
     $optionalDockerBuildArgs = "--no-cache"
 }
 
-$dockerRepo = "microsoft/dotnet-nightly"
 $dirSeparator = [IO.Path]::DirectorySeparatorChar
 $repoRoot = Split-Path -Parent $PSScriptRoot
+$dockerRepo = (Get-Content "${repoRoot}${dirSeparator}manifest.json" | ConvertFrom-Json).DockerRepo
```

Comment:
When I wrote that, I was thinking it could replace `$dirSeparator`, but I see now it's used for things that combine/join can't do. I'm fine using `$dirSeparator` here, too--either way.